Library/Formula/powerline.rb

```diff
@@ -0,0 +1,27 @@
+# Documentation: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md
+#                http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
+
+class Powerline < Formula
+  desc "Powerline is a statusline plugin for vim, and provides statuslines and prompts for several other applications, including zsh, bash, tmux, IPython, Awesome and Qtile."
+  homepage "https://github.com/powerline/powerline"
+  url "https://github.com/powerline/powerline/archive/2.3.tar.gz"
+  version "2.3"
+  sha256 "c65762df4733afed585af722978d24bc555fbbabb8ae707e4e525dff6412de5f"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  def install
+      system "python", *Language::Python.setup_install_args(prefix)
+
+      ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+      system "python", *Language::Python.setup_install_args(libexec)
```

Comment:
`setup.py` is being run twice, once here and above at line 15.  Is that necessary?